### PR TITLE
Rebrand to EnsonoBuild

### PR DESCRIPTION
**Pull request by russellseymour**
## 📲 What

Rebranded the Independent Runner module to EnsonoBuild
Updated the style of the documentation.

## 🤔 Why

Amido has been part of Ensono for a long time now and so components should be branded as such.

## 🛠 How

Renamed AmidoBuild to EnsonoBuild.
Updated the references within the module to state EnsonoBuild

## 👀 Evidence

Artifacts - https://dev.azure.com/amido-dev/Amido-Stacks/_build/results?buildId=32202&view=artifacts&pathAsName=false&type=publishedArtifacts

## 🕵️ How to test

Notes for QA

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,5 @@ docs/reference
 
 # Used to store Pester exit code to detect failure
 .PesterErrorCode
+
+.vscode
```
Comment (ElvenSpellmaker) [inline .gitignore:10]: newline

File: build/taskctl/tasks.yaml [hunk 2]
```diff
@@ -5,31 +5,39 @@ tasks:
...
   _docs:
     description: Build Docs
     context: docsenv
     command:
-      - /app/build/scripts/Build-Help.ps1
-      - Build-Documentation -BasePath /app -Pdf -Title "Independent Runner" -AttributeFile /app/build/config/pdf-config.ps1
-      - Build-Documentation -BasePath /app -MD -MDX
+      # - /app/build/scripts/Build-Help.ps1
+      # - Build-Documentation -BasePath /app -Pdf -Title "Independent Runner" -AttributeFile /app/build/config/pdf-config.ps1
+      # - Build-Documentation -BasePath /app -MD -MDX
```
Comment (ElvenSpellmaker) [inline build/taskctl/tasks.yaml:21]: Is this deliberately commented out?

File: docs/styles/base-theme.yml [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+    font_style: $heading_font_style
+admonition:
+  column_rule_color: $base_border_color
+  column_rule_width: $base_border_width
+  padding: [0, $horizontal_rhythm, 0, $horizontal_rhythm]
+  #icon:
+  #  tip:
+  #    name: far-lightbulb
+  #    stroke_color: 111111
+  #    size: 24
```
Comment (ElvenSpellmaker) [inline docs/styles/base-theme.yml:164]: Do we need this comment?

File: docs/styles/base-theme.yml [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
+  font_style: bold
+  # h1 is used for part titles (book doctype) or the doctitle (article doctype)
+  h1_font_size: floor($base_font_size * 2.6)
+  # h2 is used for chapter titles (book doctype only)
+  h2_font_size: floor($base_font_size * 2.15)
+  h3_font_size: round($base_font_size * 1.7)
+  h4_font_size: $base_font_size_large
+  h5_font_size: $base_font_size
+  h6_font_size: $base_font_size_small
+  #line_height: 1.4
```
Comment (ElvenSpellmaker) [inline docs/styles/base-theme.yml:108]: Needed?

File: docs/styles/theme.yml [hunk 5]
```diff
@@ -41,5 +41,15 @@ table:
...
+
+role:
+  highlight:
+    background-color: #fe6e69
+  notes:
+    background_color: #6941eb
+    font_color: #00ff00
+
+heading:
+  font-color: #6941eb
```
Comment (ElvenSpellmaker) [inline docs/styles/theme.yml:55]: Newline

File: docs/styles/base-theme.yml [hunk 6]
```diff
@@ -0,0 +1,286 @@
...
+  border_width: $base_border_width
+  margin_top: $vertical_rhythm * 0.5
+  margin_bottom: $vertical_rhythm * 1.5
+description_list:
+  term_font_style: bold
+  term_spacing: $vertical_rhythm / 4
+  description_indent: $horizontal_rhythm * 1.25
+outline_list:
+  indent: $horizontal_rhythm * 1.5
+  #marker_font_color: 404040
```
Comment (ElvenSpellmaker) [inline docs/styles/base-theme.yml:243]: Needed?

File: docs/styles/base-theme.yml [hunk 7]
```diff
@@ -0,0 +1,286 @@
...
+    stripe_background_color: F9F9F9
+  foot:
+    background_color: F0F0F0
+toc:
+  indent: $horizontal_rhythm
+  line_height: 1.4
+  dot_leader:
+    #content: ". "
+    font_color: A9A9A9
+    #levels: 2 3
```
Comment (ElvenSpellmaker) [inline docs/styles/base-theme.yml:264]: Needed?

File: docs/styles/base-theme.yml [hunk 8]
```diff
@@ -0,0 +1,286 @@
...
+    border_bottom_width: $base_border_width * 2.5
+  body:
+    stripe_background_color: F9F9F9
+  foot:
+    background_color: F0F0F0
+toc:
+  indent: $horizontal_rhythm
+  line_height: 1.4
+  dot_leader:
+    #content: ". "
```
Comment (ElvenSpellmaker) [inline docs/styles/base-theme.yml:262]: Needed?

File: docs/styles/base-theme.yml [hunk 9]
```diff
@@ -0,0 +1,286 @@
...
+  big:
+    font_size: $base_font_size_large
+  small:
+    font_size: $base_font_size_small
+  subtitle:
+    font_color: 999999
+    font_size: 0.8em
+    font_style: normal_italic
+# FIXME vertical_rhythm is weird; we should think in terms of ems
+#vertical_rhythm: $base_line_height_length * 2 / 3
```
Comment (ElvenSpellmaker) [inline docs/styles/base-theme.yml:69]: Needed? Part of the Fix comment or?

File: docs/styles/base-theme.yml [hunk 10]
```diff
@@ -0,0 +1,286 @@
...
+  #font_size: 11.25
+  #line_height_length: 18
+  #font_size: 11.2
+  #line_height_length: 16
+  font_size: 10.5
+  #line_height_length: 15
+  # correct line height for Noto Serif metrics
+  line_height_length: 12
+  #font_size: 11.25
+  #line_height_length: 18
```
Comment (ElvenSpellmaker) [inline docs/styles/base-theme.yml:46]: Various commented lines here, needed?

**Review by ElvenSpellmaker — COMMENTED** [review]
Generally looks good, just a couple of missing newlines, and some commented out bits, which I'm just clarifying if they're needed or not.

Nice one, thanks for this.

File: .gitignore (hunk 1 above)
Comment (russellseymour) [reply .gitignore:10]: Added as requested

File: build/taskctl/tasks.yaml (hunk 2 above)
Comment (russellseymour) [reply build/taskctl/tasks.yaml:21]: Removed

File: docs/styles/theme.yml (hunk 5 above)
Comment (russellseymour) [reply docs/styles/theme.yml:55]: Added

File: docs/styles/base-theme.yml (hunk 10 above)
Comment (russellseymour) [reply docs/styles/base-theme.yml:46]: Removed obsolete comments

File: docs/styles/base-theme.yml (hunk 9 above)
Comment (russellseymour) [reply docs/styles/base-theme.yml:69]: Removed and not part of fix

File: docs/styles/base-theme.yml (hunk 4 above)
Comment (russellseymour) [reply docs/styles/base-theme.yml:108]: Removed

File: docs/styles/base-theme.yml (hunk 3 above)
Comment (russellseymour) [reply docs/styles/base-theme.yml:164]: Removed

File: docs/styles/base-theme.yml (hunk 6 above)
Comment (russellseymour) [reply docs/styles/base-theme.yml:243]: Removed

**Review by ElvenSpellmaker — APPROVED** [review]

**Comment by ElvenSpellmaker** [comment]
Noice, ty

**Pull request merged**